audio-worklet/basic/js/bit-crusher.js

```diff
@@ -1,40 +1,46 @@
+/**
+ * @class BitCrusher
+ * @extends AudioWorkletProcessor
+ *
+ * This is an adaptation of: 
+ * https://webaudio.github.io/web-audio-api/#the-bitcrusher-node
+ */
+
+
 registerProcessor('bit-crusher', class BitCrusher extends AudioWorkletProcessor {
 
-  static get parameterDescriptors () {
-    return [{
-      name: 'bitDepth',
-      defaultValue: 12,
-      minValue: 1,
-      maxValue: 16 
-    }, {
-      name: 'frequencyReduction',
-      defaultValue: 0.5,
-      minValue: 0,
-      maxValue: 1
-    }];
+  static get parameterDescriptors() {
+    return [
+      {name: 'bitDepth', defaultValue: 12, minValue: 1, maxValue: 16}, {
```

Comment:
Is this clang-format?  If so, I really, Really, REALLY hate how this is formatted.